src/doc/reference.md

```diff
@@ -3565,7 +3565,7 @@ Each instance of a trait object includes:
    implementation (i.e. a function pointer).
 
 The purpose of trait objects is to permit "late binding" of methods. A call to
-a method on a trait object is only resolved to a vtable entry at compile time.
+a method on a trait object is only resolved to a vtable entry at runtime.
```

Comment:
I think I would probably say something like "Calling a method on a trait object results in virtual dispatch at runtime: that is, a function pointer is loaded from the trait object vtable and invoked indirectly."